iac/network/scripts/bootnode_startup.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+set -e
+
+# Installs required dependencies for running an Aztec Bootnode, creates a start script for starting the bootnode and a systemd entry
+
+# From terraform
+LOCATION="${LOCATION}"
+SSH_USER="${SSH_USER}"
+PUBLIC_IP="${PUBLIC_IP}"
+P2P_PORT="${P2P_PORT}"
+PEER_ID_PRIVATE_KEY="${PEER_ID_PRIVATE_KEY}"
+DATA_STORE_MAP_SIZE_KB="${DATA_STORE_MAP_SIZE_KB}"
+L1_CHAIN_ID="${L1_CHAIN_ID}"
+NETWORK_NAME="${NETWORK_NAME}"
+TAG="${TAG}"
+
+# Update system packages
+echo "Updating system packages..."
+sudo apt update -y && sudo apt upgrade -y
+
+# Install Docker
+echo "Installing Docker..."
+sudo apt install -y docker.io
+sudo systemctl start docker
+sudo systemctl enable docker
+sudo usermod -aG docker $SSH_USER
+sudo systemctl restart docker
+
+# Ensure Docker starts on reboot
+echo "Configuring Docker to start on boot..."
+sudo systemctl enable docker.service
+sudo systemctl enable containerd.service
+
+if [ "$LOCATION" = "GCP" ]; then
+# Install Google Cloud Ops Agent for logging
+echo "Installing Google Cloud Ops Agent..."
+curl -sSO https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh
+sudo bash add-google-cloud-ops-agent-repo.sh --also-install
+
+# Cloud Logging Configuration (Ops Agent)
+echo "Configuring Cloud Logging with Ops Agent..."
+cat <<EOF | sudo tee /etc/google-cloud-ops-agent/config.yaml
+logging:
+  receivers:
+    docker_logs:
+      type: files
+      include_paths:
+        - /var/lib/docker/containers/*/*.log
+
+  service:
+    pipelines:
+      default_pipeline:
+        receivers: [docker_logs]
+
+
+metrics:
+  receivers:
+    hostmetrics:
+      type: hostmetrics
+  service:
+    pipelines:
+      default_pipeline:
+        receivers: [hostmetrics]
+EOF
+
+# Enable and start Ops Agent
+sudo systemctl enable google-cloud-ops-agent
+sudo systemctl restart google-cloud-ops-agent
+elif [ "$LOCATION" = "AWS" ]; then
+  echo "Placeholder for AWS instances"
+fi
+
+# Create a start script to be run by systemd
+CONTAINER_NAME="aztec-bootnode"
+REPO=aztecprotocol
+IMAGE=aztec
+LOG_LEVEL=verbose
+
+cat <<EOF > /home/$SSH_USER/tag.sh
+#!/bin/bash
+export TAG=$TAG
+EOF
+chmod +x /home/$SSH_USER/tag.sh
+
+cat << 'EOF' > /home/$SSH_USER/start.sh
+#!/bin/bash
+source ./tag.sh
+echo "Starting bootnode container..."
+JSON=$(curl -s http://static.aztec.network/$NETWORK_NAME/bootnodes.json)
+export BOOTSTRAP_NODES=$(echo "$JSON" | jq -r '.bootnodes | join(",")')
```

Comment:
Does `jq` ship by default on ubuntu 22 or do we have to install it?

Reply:
Will add an install command to be sure